.gitignore

```diff
@@ -1,3 +1,4 @@
 build/
+deployments/
```

Comment:
We shouldn't ignore deployments right? This is where the network information gets saved AFAIK.

Reply:
But since addresses are deterministic they can be generated on prepack just like the build folder.

Reply:
:exploding_head: Great point.